app/Models/Team.php

```diff
@@ -88,6 +89,7 @@ class Team extends Model
     use HasRelationshipObservables;
     use Notifiable;
     use SoftDeletes;
+    use Searchable;
```

Comment:
Does this model need to have ranking rules explicitly defined?

Reply:
I couldn't think of any that would be useful.

Reply:
![image](https://user-images.githubusercontent.com/5790137/126579915-4bcbd383-f87c-4fb4-9089-6b965e0a6b7b.png)

It'd be nice if they appeared alphabetically

Reply:
Add `attendance_count`